engine/src/main/scala/cromwell/webservice/metadata/MetadataBuilderActor.scala

```diff
@@ -172,17 +172,17 @@ object MetadataBuilderActor {
     } mapValues { md => JsArray(md.toVector.sortBy(_.index) flatMap { _.metadata }) }
 
     val wrappedCalls = JsObject(Map(WorkflowMetadataKeys.Calls -> JsObject(callsMap)))
-
-    JsObject(foldedWorkflowValues.fields ++ wrappedCalls.fields)
+    val callData = if (callsMap.isEmpty && !includeCallsIfEmpty) Nil else wrappedCalls.fields
```

Comment:
No it shouldn't, the only way the behavior changes is if the `includeCallsIfEmpty` flag is set and it's only set on the workflow outputs code path.